src/routes/InstanceDetailsPage/InstanceDetailsPage.js

```diff
@@ -59,27 +53,19 @@ function InstanceDetailsPage() {
     <div>
       <PageHeader>
         <Flex direction={{ default: 'column' }}>
-          <FlexItem>
-            <Breadcrumb>
-              <BreadcrumbItemLink to="/instances">
-                ACS instances
-              </BreadcrumbItemLink>
-              <BreadcrumbItem isActive>{instance.name}</BreadcrumbItem>
-            </Breadcrumb>
-          </FlexItem>
```

Comment:
The one thing we do lose here is having the `instance.name` be the last item in a breadcrumb list. We may be able to get this into the breadcrumbs that the chrome provides, but that will take some research beyond the scope of the PF6 change.

IMO I don't think it should block the upgrade, and honestly doesn't feel like a strong priority to replace after the fact either.

Reply:
seems like a common enough use case for it to exist within a breadcrumb -- i wonder what the logic is of not having it in there. 🤔

Reply:
I suspect that the breadcrumbs that the console UI automatically provides are based on our navigation structure. The breadcrumbs that are visible always correspond to what we have in the left navigation, and the individual instance name isn't part of that.

We might be able to retain this with some research + work, but IMO it isn't necessary since we display the instance name in a heading right below the breadcrumbs.